parsing/pprintast.ml

```diff
@@ -1102,7 +1102,7 @@ and class_expr ctxt f x =
         pp f "%a%a"
           (fun f l-> if l <>[] then
               pp f "[%a]@ "
-                (list (core_type ctxt) ~sep:",") l) l
+                (list (core_type ctxt) ~sep:",@ ") l) l
```

Comment:
Rather than `@ `, `@,` would be closer to the current code and the code that you have spontaneously written as test. Also in cases of line break, it is probably better to keep the type parameter aligned with the opening `[` by opening a box.

Reply:
I added a hov box, but not is seems that '@ ' is better, because with '@,' I'm getting 
```
[ x,
  x,
  x,x,x ]
```
But with '@ ' we get 
```
[ x,
  x,
  x, x, x ]
```
P.S. The patch has been updated, but GitHub doesn't mark this conversation as outdated. I don't know why